assets/src/design-system/components/contextMenu/stories/index.js

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { action } from '@storybook/addon-actions';
+import { boolean } from '@storybook/addon-knobs';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import ContextMenu from '..';
+import { DarkThemeProvider } from '../../../storybookUtils';
+
+const items = [
+  { label: 'Copy', shortcut: '⌘ X' },
+  { label: 'Paste', shortcut: '⌘ C' },
+  { label: 'Delete', shortcut: 'DEL' },
+  { label: 'Send to back', shortcut: '⌥ ⌘ [', separator: 'top' },
+  { label: 'Send backward', shortcut: '⌘ [', disabled: true },
+  { label: 'Bring forward', shortcut: '⌘ ]', disabled: true },
+  {
+    label: 'Bring to front',
+    shortcut: '⌥ ⌘ [',
+    disabled: true,
+    separator: 'bottom',
+  },
+  { label: 'Copy style', shortcut: '⌥ ⌘ C' },
+  { label: 'Paste Style', shortcut: '⌥ ⌘ V' },
+  { label: 'Clear text styles' },
+  { label: 'Add style to "Saved styles"' },
+  { label: 'Add color to "Saved colors' },
+];
```

Comment:
Would it be better just to require an `id` on each item in the data getting passed in vs generating one at run time? probably doesn't matter all too much  either way perf wise cus the lists shouldn't be changing frequently or have large subtrees of children.

Reply:
Toatlly - I'm not sure which I prefer. I think for initial usage I like not having `id` be a field that has to be defined by the dev. Less to deal with 🤷 

I like both options, but maybe we keep it like this for now?